views/chorals.pug

```diff
@@ -30,3 +30,9 @@ block content
           td
             button.btn.btn-danger.delete-choral-btn(data-choral-id=choral.choralId)
               span.glyphicon.glyphicon-trash
+
+    script.
+      $('.choral-row').on("click", function() {
+        document.location = $(this).data('href');
```

Comment:
i can put in (cursor: pointer) but i'm not sure how to make a table row a link. I guess a button would be more obvious for users. Maybe keep this, and make a button?